### PR TITLE
Linear Regression from scratch with example has been implemented

### LINKED ISSUE #264: Implementation of Linear regression in Python
## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

(Write your answer here.)
Linear regression model built from scratch for easy understanding of concepts

## Motivation

(Please outline the motivation for the proposal.)
It is vital to understand the basics of machine learning basics

## Pitch

(Please explain why this feature should be implemented and how it would be used.)
Linear regression - definition
Gradient descent 
Learning rate
All will be explained along with the code

### PR DESCRIPTION
**Pull request by shrishiva16**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

(Write your answer here.)

### Description

(Write your answer here.)
Implemented Linear Regression from scratch in python using simple dataset
Fixes #264 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Review by photon149 — CHANGES_REQUESTED** [review]
@shrishiva16 Great work. Just a few changes which need to be addressed : 
- Add a README file , with output SS and a basic working of functions implemented.
- If Temp file not mandatory then delete the same.

**Review by photon149 — APPROVED** [review]
Looks good to me.
@Kushal997-das @prakhar3062 have a look once.

**Review by Kushal997-das — APPROVED** [review]
